packages/react-liveness/src/components/FaceLivenessDetector/service/utils/liveness.ts

```diff
@@ -305,7 +307,7 @@ export function getFaceMatchStateInLivenessOval(
     ovalBoundingBox
   );
 
-  const intersectionThreshold = OvalIouThreshold;
+  const intersectionThreshold = 0.6;
```

Comment:
This should come from the server, this was hardcoded temporarily. Please track this with the service team to update before this goes out.

Reply:
Will check with Sai